pkg/storage/gcs.go

```diff
@@ -123,6 +123,18 @@ func (s *gcsStorage) FileExists(ctx context.Context, name string) (bool, error)
 	return true, nil
 }
 
+// Open a Reader by file name
+func (s *gcsStorage) Open(ctx context.Context, name string) (ReadSeekCloser, error) {
+	// TODO, implement this if needed
+	panic("Unsupported Operation")
+}
+
+// WalkDir traverse all the files in a dir
+func (s *gcsStorage) WalkDir(ctx context.Context, fn func(string, int64) error) error {
```

Comment:
please document that the first argument of `fn` is the full path to be passed into `Open()` and the second argument is the file size.